dist/index.bundle.js

```diff
@@ -1 +1 @@
\ No newline at end of file
+(()=>{const e=document.getElementById("form"),t=document.querySelector(".taskadder"),n=document.querySelector(".sync"),d=document.querySelector(".enter-icon"),a=document.getElementById("delcompleted");let i;const c=e=>{e.forEach((e=>{document.getElementById("duties").appendChild(document.createElement("li")).setAttribute("id",e.index);const t=document.getElementById(e.index);t.classList.add("task","draggable"),t.appendChild(document.createElement("input")).setAttribute("id",`${e.index}-checkbox`);const n=document.getElementById(`${e.index}-checkbox`);n.setAttribute("type","checkbox"),n.classList.add("checkbox"),t.appendChild(document.createElement("p")).setAttribute("id",`${e.index}-description`);const d=document.getElementById(`${e.index}-description`);d.classList.add("description"),d.innerText=e.description,t.appendChild(document.createElement("i")).setAttribute("id",`${e.index}-drag`);const a=document.getElementById(`${e.index}-drag`);a.classList.add("fas","fa-ellipsis-v","drag-btn"),t.appendChild(document.createElement("i")).setAttribute("id",`${e.index}-trash`),document.getElementById(`${e.index}-trash`).classList.add("far","fa-trash-alt","trash-btn"),function(e){const t=e.parentElement;e.addEventListener("mousedown",(()=>{t.setAttribute("draggable",!0)}))}(a)})),function(e){e.forEach((t=>{const n=document.getElementById(t.index);n.addEventListener("dragstart",(e=>{e.dataTransfer.setData("index",t.index)})),n.addEventListener("drop",(d=>{const a=d.dataTransfer.getData("index"),i=t.index,c=e[a],s=e[i];e[a]=s,e[i]=c,c.index=i,s.index=a,n.setAttribute("draggable",!1),window.localStorage.setItem("tasklist",JSON.stringify(e)),window.location.reload()})),n.addEventListener("dragover",(e=>{e.preventDefault()}))}))}(e),function(e){e.forEach((t=>{const n=document.getElementById(`${t.index}-checkbox`);n.addEventListener("change",(()=>{n.checked?(t.completed=!0,window.localStorage.setItem("tasklist",JSON.stringify(e))):(t.completed=!1,window.localStorage.setItem("tasklist",JSON.stringify(e)))}))}))}(e),function(e){e.forEach((e=>{const t=document.getElementById(`${e.index}-checkbox`);!0===e.completed?t.checked=!0:t.checked=!1}))}(e),edit(e),removetask(e)};e.addEventListener("submit",(e=>{e.preventDefault(),""!==t.value&&new Duty(t.value).push(i)})),d.addEventListener("click",(e=>{e.preventDefault(),""!==t.value&&new Duty(t.value).push(i)})),a.addEventListener("click",(e=>{e.preventDefault(),removecompleted(i)})),n.addEventListener("click",(e=>{e.preventDefault(),removeAll(i)})),document.addEventListener("load",void(JSON.parse(localStorage.getItem("tasklist"))?(i=JSON.parse(localStorage.getItem("tasklist")),c(i)):(i=[],c(i))))})();
```

Comment:
- Kindly fix error that you will find into your chrome console. There are issues related into your index.js.